modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/analytics/base/AnalyticsBaseTestCase.java

```diff
@@ -55,13 +55,11 @@ public void disableAnalytics() throws Exception {
     private void changeConfiguration() {
 
         log.info("Changing identity.xml file to enable analytics");
```

Comment:
Log message needs to fix

Reply:
Fixed - https://github.com/wso2/product-is/pull/7059